src/common/views/opds.ts

```diff
@@ -78,10 +87,34 @@ export interface OpdsAuthView {
     oauthRefreshUrl: string;
 }
 
+export interface IOPDSPropertiesView {
+    numberOfItems?: number | undefined;
+    priceValue?: number | undefined;
+    priceCurrency?: OPDSCurrencyEnum | undefined;
+    holdTotal?: number | undefined;
+    holdPosition?: number | undefined;
+    copyTotal?: number | undefined;
+    copyAvailable?: number | undefined;
+    availabilityState?: OPDSAvailabilityEnum | undefined;
+    availabilitySince?: string | undefined;
+    availabilityUntil?: string | undefined;
```

Comment:
Just a thought: why `?` optional property as well as `undefined` union type?

Reply:
not the same thing , but in this case is useless

Reply:
Because TypeScript `strictNullCheck` is not currently used, we have these kinds of inconsistencies. So, for now, let's accept this redundant syntax to help with Intellisense in Visual Studio Code.